Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -92,6 +101,20 @@ bool fetchInstructionsMode::ReceiveTeleopInstructions(attitudeManager* attitudeM
     return true;
 }
 
+bool fetchInstructionsMode:: isArmed()
+{
+    //Range of PPM percentage to be considered ARMED is 95-100%
+    const uint8_t MIN_ARM_VALUE = 50;
+    if (_TeleopInstructions.PPMValues[ARM_DISARM_CHANNEL_INDEX] >= MIN_ARM_VALUE)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
```

Comment:
See Dhruv's previous comments
```suggestion
    bool retVal = false;
    if (_TeleopInstructions.PPMValues[ARM_DISARM_CHANNEL_INDEX] >= MIN_ARM_VALUE)
    {
        retVal = true;
    }
    return retVal;
```